### ISSUE TITLE
Games being removed from the wiki

### ISSUE THREAD

**Issue by Z4ndyz** [Nukem9/dlssg-to-fsr3#411]
https://github.com/Vladzor87do  This user seems to be removing games from the wiki that have unofficial implementation. All of his changes should be verified and removed/reverted

**Comment by TimmyP777** [comment]
This entire thing is ridiculous. If you talk about this mod and its real benefits, almost anywhere, your content will be hidden or removed. Simply say something like "removes the need to go from 3000 to 4000" or "DLSS3 for 2000/3000 series" on a GN\DF video, or reddit comment on pcgaming\pcmr and it will be shadow muted by someone above a mod! Sound ridiculous I know, but I have proof.

Go ahead and test. When you realize Im not joking, can someone explain?

**Comment by Vladzor87do** [comment]
> https://github.com/Vladzor87do This user seems to be removing confirmed working games(not sure if intentional or by mistake since some of them are in the dropdown list but they're removed from the wiki home https://github.com/Nukem9/dlssg-to-fsr3/wiki/Game-Compatibility-List) that have unofficial implementation. All of his changes should be verified and removed/reverted if they aren't fixed

yes, i am removing game with are using only PureDark mods and not Nukem9 mod,  PureDark mods and LukeFZ mods have nothing to do with DLSS-TO-FSR3 mod

**Comment by LeBlackMeow** [comment]
> yes, i am removing games wich are using only PureDark mods and not Nukem9 mod, PureDark mods and LukeFZ mods have nothing to do with DLSS-TO-FSR3 mod
> 
> I have removed RDR2 and Elden Ring entries since they have inofficial PureDark FSR3 impementations, but I have added GTA V since DLSS-to-FSR3 is not working without PureDark mod
> 
> but thx for remind me to remove unnecessary pages

It is possible to add a change log/historic? So people know what change had been made and why?

**Comment by Z4ndyz** [comment]
Aight makes some sense now btw i wasn't trying to be offensive i just found it weird that entries were removed and i couldn't find a reason anywhere.

**Comment by Vladzor87do** [comment]
> Aight makes some sense now btw i wasn't trying to be offensive i just found it weird that entries were removed and i couldn't find a reason anywhere.

now there is an explanation for this ;)
